index.html

```diff
@@ -1725,6 +1727,69 @@ <h3>Validity Period</h3>
 }
         </pre>
 
+        <h4>Representing Time</h4>
+
+        <p>
+When representing time values, implementers are urged to understand that times
+are not implemented in a uniform manner throughout the world. For example:
+        </p>
+
+        <ul>
+          <li>
+The observation of daylight savings time does not start or end on the same day
+and time, if at all, between various regions.
+          </li>
+          <li>
+The adoption of the Gregorian calendar did not happen in a single year, it was
+phased in over many years across various societies.
+          </li>
+          <li>
+Leap seconds might not be taken into account in all software systems, especially
+for dates and times that precede the introduction of the leap second.
+          </li>
+        </ul>
+
+        <p>
+These are just a few examples that illustrate that whether or not a time is seen
+as valid can be based on cultural norms. In other words, some dates and times
+exist for some cultures, while they do not exist for other cultures.
+        </p>
+
+        <p>
+This specification attempts to increase the number of universally recognized
+combinations of dates and times, and reduce the potentials for
+mis-interpretation of time values, by utilizing the
+<code>dateTimeStamp</code> construction first established by the [<a
+data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification. In
+order to reduce mis-interpretations between different cultures, all times are
+specified as offsets against Universal Coordinated Time (UTC).
+        </p>
+
+        <p>
+Implementers that desire to check <code>dateTimeStamp</code> values for validity
+can use the regular expression provided below, which is reproduced from the [<a
+data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification for the
+purposes of convenience. For the avoidance of doubt, the regular expression in
```

Comment:
```suggestion
data-cite="XMLSCHEMA11-2#dateTimeStamp">XMLSCHEMA11-2</a>] specification for
convenience. To avoid doubt, the regular expression in
```